lib/facter/fpm_pools.rb

```diff
@@ -0,0 +1,9 @@
+Facter.add(:fpm_pools) do
+  confine :osfamily => 'Debian'
+
+  setcode do
+    Dir.glob('/etc/php5/fpm/pool.d/*.conf').each do |pool_config|
```

Comment:
I know this is only on debian, but if merge that, i think the should be this feature on all distros.